### ISSUE TITLE
Training - Incorrect creation of Eval testset

### ISSUE THREAD

**Issue by marcotrombetti** [ModernMT/MMT#110]
All files on my machine.

I run an evaluate
```
./mmt evaluate -b b1_sprint3 
```
While I was doing human evaluate I found this
```
ORIGINAL: 1192	en	Modem Advanced
HUMAN   : 1192	it	Modem Avanzate
MMT     : 1192	it	Modem avanzate
GT      : 1192	it	Modem avanzata
```

Unfortunately this is present in the train corpus just once, so I should not appear in the test set.
```
ubuntu@ip-172-30-0-162:~/mmt$ grep -i "Modem Advanced" ../benchmark-1.0/data/train/*.en
../benchmark-1.0/data/train/taus-undefined-milengo-10584.en:Modem Advanced
```

**Comment by davidecaroselli** [comment]
This is exactly the behavior we agreed to apply to the automatic splitting: if the training set contains duplicates, it is possible that you can have overlapping between training an test sets.

**Comment by davidecaroselli** [comment]
Removed "invalid label", I'm checking the data. Sorry for the label, I though the problem was different.

**Comment by davidecaroselli** [comment]
Ok, so in the folder `benchmark-1.0/data/train/*.en` you have all the original training corpus. The one the user provides.
The automatic splitting reads those files and selects a portion to use as dev and test set. So it's perfectly normal to find a test sentence in the **original** corpora; the corpora used during training **is not** exactly the original , but of course the one where MMT has removed the sentences used for test and training.

You will find **all** the test (and dev) sentences in the original corpora. But you won't find them, if they are unique in the training corpus, in the MMT models.
